src/main/java/uk/gov/hmcts/reform/et/syaapi/service/CaseService.java

```diff
@@ -198,27 +198,27 @@ public CaseDetails submitCase(String authorization,
             caseDetails.getData().get("ethosCaseReference").toString());
         List<PdfDecodedMultipartFile> acasCertificates = pdfService.convertAcasCertificatesToPdfDecodedMultipartFiles(
             caseData, acasService.getAcasCertificatesByCaseData(caseData));
-        PdfDecodedMultipartFile casePdfFile =
+        List<PdfDecodedMultipartFile> casePdfFiles =
             pdfService.convertCaseDataToPdfDecodedMultipartFile(caseData);
         caseDetails.getData().put("ClaimantPcqId", caseData.getClaimantPcqId());
         caseDetails.getData().put("documentCollection",
                                   caseDocumentService
                                       .uploadAllDocuments(authorization,
                                                           caseRequest.getCaseTypeId(),
-                                                          casePdfFile,
+                                                          casePdfFiles,
                                                           acasCertificates));
 
         triggerEvent(authorization, caseRequest.getCaseId(), UPDATE_CASE_SUBMITTED, caseDetails.getCaseTypeId(),
                      caseDetails.getData());
-        notificationService
-            .sendSubmitCaseConfirmationEmail(
-                notificationsProperties.getSubmitCaseEmailTemplateId(),
-                caseData.getClaimantType().getClaimantEmailAddress(),
-                caseRequest.getCaseId(),
-                caseData.getClaimantIndType().getClaimantFirstNames(),
-                caseData.getClaimantIndType().getClaimantLastName(),
-                caseDetails.getId() == null ? "case id not found" : caseDetails.getId().toString(),
-                notificationsProperties.getCitizenPortalLink());
+//        notificationService
```

Comment:
Probably this one should be uncommented in commit?

Reply:
Correct, we will add this back in

Reply:
done